lib/parser.js

```diff
@@ -66,6 +66,15 @@ const args = [
           'on this server'
   }],
 
+  [['-ka', '--keep-alive-timeout'], {
+    required: false,
+    default: null,
+    dest: 'keepAliveTimeout',
+    type: 'int',
+    help: 'Number of seconds the Appium server should apply as the connection timeout for all ' +
```

Comment:
https://stackoverflow.com/questions/49704708/what-is-a-connection-timeout-during-a-http-request

Although, keep alive is https://nodejs.org/api/http.html#http_server_keepalivetimeout

Reply:
@mykola-mokhnach I see. the main point of my change has to do with the connection timeout aspect. What would you want to see changed here to make things clear, if anything?

Reply:
The code is fine. I just thought about tuning the help string to avoid possible misunderstandings